src/models/synced_service/service_object/service_object.ts

```diff
@@ -2,10 +2,14 @@ export class ServiceObject {
   id: number | string;
   name: string;
   type: string;
+  projectId: number | string | null
+  syncCustomFieldValue: number | string | null
 
-  constructor(id: number | string, name: string, type: string) {
+  constructor(id: number | string, name: string, type: string, projectId: number | string | null = null, syncCustomFieldValue: number | string | null = null) {
     this.id = id;
     this.name = name;
     this.type = type;
+    this.projectId = projectId
```

Comment:
Co zde znamená projectId?

Reply:
Id projketu v případě, že je to úkol. Pro Jira jsem to potřeboval. Přidán komentář